### PR TITLE
Updates documentation for runjob mgmt command

### PR DESCRIPTION
**Pull request by joewesch**
# Closes: #<ISSUE NUMBER GOES HERE>
# What's Changed

Updates the documentation regarding the `nautobot-server runjob` management command.

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

File: nautobot/docs/user-guide/administration/tools/nautobot-server.md [hunk 1]
```diff
@@ -688,19 +688,24 @@ Done.
 
 `nautobot-server runjob [job]`
```
Comment (glennmatthews) [inline nautobot/docs/user-guide/administration/tools/nautobot-server.md:689]:
```suggestion
`nautobot-server runjob --username <username> [--local] [--data <data>] <job>`
```

**Review by glennmatthews — APPROVED** [review]
Thanks!

File: nautobot/docs/user-guide/administration/tools/nautobot-server.md [hunk 2]
```diff
@@ -686,21 +686,26 @@ Done.
...
 ### `runjob`
 
-`nautobot-server runjob [job]`
+`nautobot-server runjob --username <username> [--local] [--data <data>] <job>`
 
-Run a job (script, report) to validate or update data in Nautobot.
-
-`--commit`  
-Commit changes to DB (defaults to dry-run if unset). `--username` is mandatory if using this argument.
+Run a job (script, report) to validate or update data in Nautobot. The Job name must be in the Python module form: `<module_name>.<JobClassName>`.
```
Comment (itdependsnetworks) [inline nautobot/docs/user-guide/administration/tools/nautobot-server.md:691]:
```suggestion
Run a job (script, report) to validate or update data in Nautobot. The Job name must be in the Python module form: `<module_name>.<JobClassName>`. You can find this under the Jobs detail view on the "Class Path" row.
--


<br class="Apple-interchange-newline">
```

Comment (joewesch) [reply nautobot/docs/user-guide/administration/tools/nautobot-server.md]: @itdependsnetworks was this part intentional? What does this add?

```
<br class="Apple-interchange-newline">
```

Comment (itdependsnetworks) [reply nautobot/docs/user-guide/administration/tools/nautobot-server.md]: It was not... by bad, I have updated the comment.

**Pull request merged**